pkgs/by-name/ya/yazi-unwrapped/update.sh

```diff
@@ -14,10 +14,10 @@ YAZI_RELEASE=$(
 
 # Get release information
 latestBuildDate=$(echo "$YAZI_RELEASE" | jq -r ".published_at")
-latestVersion=$(echo "$YAZI_RELEASE" | jq -r ".tag_name")
+tagName=$(echo "$YAZI_RELEASE" | jq -r ".tag_name")
 
 latestBuildDate="${latestBuildDate%T*}" # remove the timestamp and get the date
-latestVersion="${latestVersion:1}"      # remove first char 'v'
+latestVersion="${tagName:1}"      # remove first char 'v'
```

Comment:
it's more readable if the comments are aligned, in my opinion

```suggestion
latestVersion="${tagName:1}"            # remove first char 'v'
```